### ISSUE TITLE
Ultrachromic backdrop configuration not properly applied

### ISSUE THREAD

**Issue by VladTheSheep** [danieladov/jellyfin-plugin-skin-manager#47]
Only one backdrop configuration is applied at the time. The filter for .backdropImage needs to be merged into one or else one of the four configurations will take priority. Only tested on Firefox.

![Screenshot from 2022-08-28 10-55-47](https://user-images.githubusercontent.com/6963524/187066040-43450916-7c26-4b7c-ba92-9d334af7dfde.png)

Can be worked around by going to Dashboard -> General, then remove the existing lines for .backdropImage in the CSS config with one line.

`.backdropImage {filter: blur(18px) saturate(120%) contrast(120%) brightness(40%);}`

**Comment by Archer-Tatsu** [comment]
Met the same problem and solved with the same method.

**Comment by stankovictab** [comment]
Reporting same problem on Chromium (Vivaldi specifically), same solution

**Comment by Mhowser** [comment]
yep had to apply this fix as well, @danieladov any chance you could look into this?

**Comment by Taparu** [comment]
Likewise same issue here. Tested firefox and chrome.
